card/spec/models/card/trash_spec.rb

```diff
@@ -201,9 +201,9 @@
 describe Card, "junction revival" do
   before do
     Card::Auth.as_bot do
-      @c = Card.create! :name=>"basicname+woot", :content=>"basiccontent"
+      @c = Card.create! name: "basicname+woot", content: "basiccontent"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.